### PR TITLE
feat(itemsapi): add itemsApi to plugins

### PR DESCRIPTION
**Pull request by Camc314**

### CODE REVIEW

**Review by MrTimscampi — CHANGES_REQUESTED** [review]
Please avoid changing the indentation of the files :wink:

You can set your editor to format them with Prettier when they save, to avoid this (If you use VSCode, you can install the Prettier extension and the workspace config in the repo should take care of it)

**Comment by Camc314** [comment]
Ahh that's much better, I thought I already had Prettier installed, but clearly not :p.

Hopefully no more indentation issues 🤞

File: plugins/itemsApi.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Context } from '@nuxt/types';
+import { ItemsApi } from '../api/api';
+import { Configuration } from '../api/configuration';
+
+declare module '@nuxt/types' {
+  interface Context {
+    $ItemsApi: ItemsApi;
```
Comment (MrTimscampi) [inline plugins/itemsApi.ts:7]: Slight casing error here, your injected object would be `$itemsApi` and not `$ItemsApi`

Comment (Camc314) [reply plugins/itemsApi.ts]: Fixed 👍

**Review by MrTimscampi — APPROVED** [review]

**Pull request merged**